tests/Visitor/ReflectionVisitorTest.php

```diff
@@ -138,7 +138,7 @@ public function isPartOfSignatureFlagProvider(): array
         ];
     }
 
-    private function getSpyVisitor(string $nodeClass): NodeVisitorAbstract
```

Comment:
could you please explain why it was removed?

Reply:
We are type-hinting against `NodeVisitorAbstract`, but then calling the `isPartOfSignature` function, which is not part of the `NodeVisitorAbstract` api. 

For me by removing the return type here, PhpStorm is also capable of detecting the `isPartOfSignature` function, which it previously wasn't.